.github/scripts/test_covering_pr.js

```diff
@@ -0,0 +1,36 @@
+const Redis = require('ioredis');
+const fs = require('fs').promises;
+
+const main = async () => {
+    const changedFiles = process.argv.slice(2)
+
+    /**
+     * Connects to a Redis Database
+     * TODO: Move these variables to GitHub secrets of our project, only admins have rights for that.
```

Comment:
Ist this TODO still for this PR or will you take care of this in the future?

Reply:
Also when these get moved to secrets they need to be changed, the history would still have them.

Reply:
I would prefer to keep some time with this working, and then later, we move it to GitHub secrets, in any case, it's read-only.
Also, we might also decide to move this Redis Instance somewhere else, for now I created it with my own account in a free tier in AWS.

Reply:
Would it be a big deal to keep this temporary Redis instance, and it's read-only user, in that first attempt?
@renner @juliogonzalez 

If we feel happy with that outcome, we can then proceed to use GitHub secrets and use a more production environment for the Redis DB.

Reply:
I am OK with this as something temporary, as it's readonly, and as long as it does not contain confidential data.

But **only** if @renner approves it (as he's our security champion :-))

As for the git history, true, but when we change to something else, I'd not say `Uyuni2023!` is a password I'd consider secure, so we'll need to change it anyway :-D

Reply:
Some more context:

- I created a completely ephemeral Redis instance in a personal account on [app.redislabs.com](http://app.redislabs.com/)
- It's a free DB, it doesn't cost me anything, it's located in an AWS Free tier, managed by RedisLabs
- We only add content there from one of our internal jenkins pipelines, and the user that have write access it's located in susemanager-ci (there it will be good to have a GitHub Secrets too, in the future)
- In the GitHub actions, we only have a read-only user for Redis, that query content based on the filename of the changes of a PR, the file is the key in that Redis Set, and as Values it returns all the tests that triggers at some point that class
- The only content stored in that Redis DB, is Java class names as keys, and cucumber feature names as values of these keys.
- Every time the pipeline https://ci.suse.de/view/Manager/view/Uyuni/job/uyuni-master-dev-acceptance-tests-code-coverage/ runs, it will override the content on that Redis DB, so we don't care if for some reason it dies.

Reply:
This might be okay for a prototype or during the course of an experiment, but from my perspective is not great to merge it like this for a temporary solution.  At least hostname, port, username and password should be configurable instead of hard-coded and stored as proper secrets, if possible. I would rather do that right away before merging, temporary solutions have a tendency to stay around longer than you intially might think.

Reply:
OK, changed :)